langchain/vectorstores/analyticdb.py

```diff
@@ -59,6 +58,14 @@ def __init__(
         self.logger = logger or logging.getLogger(__name__)
         self.__post_init__()
 
+        try:
+            from sqlalchemy.engine import Row
```

Comment:
i think this will need to be imported in whatever methods use this as well, it isnt in the global namespace